nvme.c

```diff
@@ -5244,8 +5244,8 @@ static int sanitize(int argc, char **argv, struct command *cmd, struct plugin *p
 	}
 
 	if (sanact == NVME_SANITIZE_SANACT_START_OVERWRITE) {
-		if (cfg.owpass > 16) {
-			fprintf(stderr, "OWPASS out of range [0-16]\n");
+		if (cfg.owpass > 15) {
```

Comment:
If `owpass` is 3 bits, how can it hold a value of 15?

Reply:
As I mentioned on commit message owpass is 4bit value

Figure 303: Sanitize – Command Dword 10

Bits [07:04]
Overwrite Pass Count (OWPASS): This field specifies the number of overwrite passes (i.e., how many times the media is to be overwritten) using the data from the Overwrite Pattern field of this command. A value of 0h specifies 16 overwrite passes. This field shall be ignored unless the Sanitize Action field is set to 011b (i.e., Overwrite).